src/llmcompressor/observers/imatrix.py

```
@@ -89,82 +110,50 @@ def _hook(mod, args):
         module._imatrix_hook = module.register_forward_pre_hook(_hook)
 
     def detach(self, module: torch.nn.Module) -> None:
-        """Remove hooks and compute / clean up importance data.
+        """Remove hooks and leave raw sum/count on module for second-pass pickup.
 
-        Case 1 – accumulators present (``_imatrix_sum``): compute importance,
-        remove the hook and accumulators, **leave** ``_imatrix_importance`` on
-        the module so the next quantization pass can use it.
+        Case 1 – accumulators present on module: leave them for next
```

Ruff (0.15.11)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)